plugins/inputs/lotus/lotus.go

```diff
@@ -296,13 +344,54 @@ func recordTipsetStatePoints(ctx context.Context, api api.FullNode, acc telegraf
 	return nil
 }
 
+func recordMpoolUpdatePoints(ctx context.Context, acc telegraf.Accumulator, newMpoolUpdate api.MpoolUpdate, receivedAt time.Time, recordedFrom peer.ID) error {
+	var updateType string
+	switch newMpoolUpdate.Type {
+	case MpoolAdd:
+		updateType = "MpoolAdd"
+	case MpoolRemove:
+		updateType = "MpoolRemove"
+	case MpoolBootstrap:
+		updateType = "MpoolBootstrap"
```

Comment:
Another thought: These strings are a little verbose to be repeated data within that schema. How about `add`,`rm`,`init`?